src/Fixer/FunctionNotation/MethodArgumentSpaceFixer.php

```diff
@@ -81,8 +93,14 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
         for ($index = $tokens->count() - 1; $index > 0; --$index) {
             $token = $tokens[$index];
 
-            if ($token->equals('(') && !$tokens[$index - 1]->isGivenKind(T_ARRAY)) {
-                $this->fixFunction($tokens, $index);
+            if ($token->equals('(')) {
+                $meaningfulTokenBeforeParenthesis = $tokens[$tokens->getPrevMeaningfulToken($index)];
+                if (!$meaningfulTokenBeforeParenthesis->isKeyword()
+                    || $meaningfulTokenBeforeParenthesis->isGivenKind(T_LIST)) {
+                    if ($this->fixFunction($tokens, $index) && $this->configuration['ensure_fully_multiline']) {
```

Comment:
One-time-variable (https://a-nickels-worth.blogspot.fr/2016/04/a-guide-to-naming-variables.html)